hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -290,20 +293,26 @@ public XceiverClientFactory getXceiverClientManager() {
     return xceiverClientManager;
   }
 
-  static boolean validateOmVersion(String expectedVersion,
+  static boolean validateOmVersion(OzoneManagerVersion expectedVersion,
```

Comment:
I think `expectedVersion` -> `minimumVersion` makes more sense with the current changes.

Reply:
done, thanks for spotting!